tools/msvs/msi/i18n/en-us.wxl

```diff
@@ -13,7 +13,7 @@
     <String Id="NativeToolsDlgDescription">Optionally install the tools necessary to compile native modules.</String>
     <String Id="NativeToolsDlgBannerBitmap">WixUI_Bmp_Banner</String>
     <String Id="NativeToolsDlgIntro">Some npm modules need to be compiled from C/C++ when installing. If you want to be able to install such modules, some tools (Python and Visual Studio Build Tools) need to be installed.</String>
-    <String Id="NativeToolsDlgInstallCheckbox">Automatically install the necessary tools. Note that this will also install Chocolatey. The script will pop-up in a new window after the installation completes.</String>
+    <String Id="NativeToolsDlgInstallCheckbox">Automatically install the necessary tools. Note that this will also install Chocolatey, Python and Visual Studio to C: drive or a required update. The script will pop-up in a new window after the installation completes.</String>
```

Comment:
Maybe:

Note that this will also install Chocolatey, Python, Visual Studio, and required Windows updates to the C: drive. The installed tools will take about 3Gb of disk space.